### ISSUE TITLE
Stuck in Loop, can't save data

### ISSUE THREAD

**Issue by weasel0x00** [KiloQubit/probe_accuracy#2]
I somehow managed to get the macro stuck in a state that it wouldn't finish, and was probing forever.  It would be useful if we could issue a M117 TEST_PROBE_ACCURACY: DONE on the console or something to convince the script to save the data it had and finish.  That doesn't seem to work though, best I can tell it doesn't come across the socket as a "response" value the script is looking for to finish.  Not familiar enough with Klipper to know what it might look like in that socket, and my basic debugging skills never saw anything to key off of.

**Comment by KiloQubit** [comment]
Thank you for your bug report.  I will try to find a way to send the script a stop signal.

**Comment by weasel0x00** [comment]
Yeah, I was thinking it might be easiest in the python script to just catch a keyboard interrupt (Ctrl-C) and then have it save the data it has at that point, generate a graph if it can, and then exiting...

**Comment by Hilpas** [comment]
Same problem here, probes forever.
